plugins/parcel/coalescing/CMakeLists.txt

```diff
@@ -22,7 +22,10 @@ function(add_coalescing_module)
         FOLDER "Core/Plugins/MessageHandler"
         )
 
-    add_definitions(-DHPX_PARCEL_COALESCING_MODULE_EXPORTS)
+    if (TARGET hpx::apex)
+      target_link_libraries(parcel_coalescing PRIVATE hpx::apex)
+    endif()
+    target_compile_definitions(parcel_coalescing INTERFACE HPX_PARCEL_COALESCING_MODULE_EXPORTS)
```

Comment:
Shouldn't this be `PRIVATE` instead of `INTERFACE`?

Reply:
Absolutely !